backend/benefit/applications/tests/test_applications_api.py

```diff
@@ -576,9 +576,10 @@ def test_application_put_edit_employee(api_client, application):
     """
     modify existing application
     """
+    new_ssn = "260778-323Y"
```

Comment:
And here let's use non-temporary SSN :)

Reply:
Did you run any tests with 2023 SSN-style as per https://dvv.fi/hetu-uudistus?

Reply:
I guess they are already tested thoroughly in the lib though 🤔

Reply:
Actually, it looks like python-stdnum doesn't support the new middle characters. Python-stdnum is well maintained, so should we make a pull request there to add the support, or rather just fix our own implementation and put it under shared code, what do you think?

I think there's no rush, we can wait python-stdnum to be updated:

_Tilanne 1/2023: Tämänhetkisen arvion mukaisesti uuden välimerkkiasetuksen mukaisia uusia henkilötunnuksen välimerkkejä annetaan ensimmäisen kerran maaliskuussa 2024._